### DISCUSSION TITLE
Deliberately Developmental Spaces follow up

### DISCUSSION THREAD

**Discussion by matthewmccarthy11** [life-itself/community#888] [Research]
This discussion post has been created as a follow up thread from the Deliberately Developmental Spaces call that took place on October 30th. If you weren't able to make it, below here are the recording and the slides: 

[Recording of the talk](https://drive.google.com/file/d/1AGe6EjPjU0OJszx-R9diV3KYX5QeOofU/view)

[Deliberately Developmental Spaces slides.pptx.pdf](https://github.com/life-itself/community/files/13220756/Deliberately.Developmental.Spaces.community.call.pptx.pdf)

In this discussion forum, we might further consider some of the things brought up in this talk, or other related topics as it relates to what a 'DDS' is, how to 'build the field', where we currently are, and any other relevant considerations. Below are a few background concepts and questions so as to consider this further. Additionally, we have had a few other github discussions as it relates to this topic, like #700  #839 , #850 and #859. 

**Definition of a DDS**: A DDS is a "physically co-located group together for a sustained period of time with a conscious engagement in multidimensional developmental praxis." (Should we keep this definition?)

**Considering the 'actors in the field'**: At this point in time, we have suggested that there are 'three kinds of actors' in the DDS field: connectors, DDS's, and participants. 

Connectors: This broad group contains the organizations who funded this report, as well as any other organizations or individuals who are concerned with connecting, funding, researching, and building the field in whatever way makes most sense. 

DDS's: Deliberately Developmental Spaces, as outlined above, are physically co-located groups together for a sustained period of time with a conscious engagement in multidimensional developmental praxis.

Participants: Young adults and youth who are interested in getting involved in a DDS.

Indeed, there can be overlap in some areas, as, for example, a 'DDS' who is 'close to the center' of the field, like, for example, ILALI, might in some ways be considered both a connector and a DDS. 

**Considering the 'field'**: We use the metaphor of a 'field' to try and make sense of how this is developing. Additionally, we are more concerned with finding the 'center', and less concerned with finding the 'boundary'. However, considering certain examples, like Harvard, which might be outside the boundary, have also proven to be meaningful as far as 'identifying the center' goes. 

**The creative community model** suggests that there are three primary 'conditions' which help enable development: safety, intentional challenge, and youth adult partnerships. 

**Potential questions**: 

How long does a DDS program take in order to have a 'substantial' or 'transformational' effect? (is it more than 2 months?)

Does it make sense to think about a rubric of development, which might help track ones own development? 

How does a DDS fit into how we consider 'education'? 

Is there an ideal organizational form to a DDS? If so, what can be said about it? 

How might youth adult partnerships be understood in a 'time of transition', when our social roles and our social structure is changing? 

Along with these questions/considerations, please feel free as well to add any recommendations to our list of DDS's.

**Comment by baouroux** [comment]
Following on from Rufus' mentioning his training with Lectica.org and their Foundations of Lectica Assessments (FOLA), I hope this is an appropriate place to post a reference to my own inquiry with a very high-level view of the different schools of adult developmental psychology, and latest research on comparing models. Lectica is in the post-Piagetian school, building on more recent work in Dynamic Skill theory from Kurt Fischer, and model of hierarchical complexity (MHC) from Michael Commons (here Rufus can give the better overview!).

In parallel there have been important developments in 'Ego' stage development, from the likes of Robert Kegan, Bill Torbert, Suzanne Cook-Greuter etc., that seems to take a more holistic approach to capturing the meaning-making patterns of an individual, and equally presenting the ego's adult pathway in a hierarchical path - leading to more and more leadership coaching referring to 'Vertical Development'. It is upon more generally the ego development models that initiatives such as the Inner Developmental Goals were launched.

I also have a particular interest in Lawrence Kohlberg's stages of moral development, and James Fowler's stages of faith (not just in religious terms, but 'other-than-self').

Jonathan Reams and Aiden Thornton for me are two important voices in an independent and thorough critique of where we are at with the contrasting methods and frameworks of development, comparing the above as 'hard-stage' models and 'soft-stage' models. Aiden has been going deep with his PhD thesis in examining if these tools are measuring different things, and their reliability. Jonathan published a review of Aiden's work and then Aiden replied - both published in the final edition of the Integral Review, and attached here (INTEGRAL REVIEW September 2023 Vol. 18, No. 1).

Just one extract from Aiden's paper:
"If there is only an approximate correspondence between ego development stages and stages of hierarchical complexity, then this implies that hierarchical integrations do not fully – or perhaps adequately – account for the transformation of performance between successive stages. This may also relitigate considerations about the logic that underpins the ego development scale and the basis for
distinguishing between adjacent stages (Cook-Greuter, 1990; Habermas, 1979; Noam, 1993)."
...that gives a flavour as to how much is still up for debate and understanding!

All this to say, that my own inquiry (having piloted and promoted Lectica in the UK from 2009 and trained as a certified Lectica coach in 2015), leads me to reflect on how the developmental models are certainly pointing to structural aspects of our psyche (e.g. the parts), but that contrasts with an aesthetic lens of the "whole".

We are in the contentious debate from people such as Nora Bateson (seemingly categorically anti-stage) and Dave Snowden (willing to consider stages, but not hierarchically), and yet I don't hear so far an exploration of the movement BETWEEN the hierarchical (parts) and the aesthetic (whole). I've had a first go at developing this inquiry in this LinkedIn post and GDoc paper: https://www.linkedin.com/posts/john-oliver-bb01a_reframing-the-controversies-in-developmental-activity-6987746415879798784-RRX_?utm_source=share&utm_medium=member_desktop

My main thrust is holding developmental psychology frameworks in the context of 'cognitive anthropology' - recognising valid hierarchical patterns of cognition in the 3rd person, but that we can gain a lot by simultaneously holding the 1st and 2nd person. From that, we can concentrate on what are the movements between the 1st, 2nd and 3rd person perspectives, versus getting stuck on the 3rd person validity debates. Developmental psychology becomes then a grammar for our future Dialogos (taking at two levels there references from John Vervaeke!). My intuition is that the 1st person "aesthetic" is the least discussed in the developmental community....and hence my own explorations in the artistic and creative experience (www.tyler.world).

How might the 'dynamics' between 1st, 2nd and 3rd person (in other words integrating the Transcendentals, or looking at the liminalities across the Integral Quadrants contribute to the developmental framework discussions and our "sensemaking" of poly crisis hyper objects?!

I'd love to hear your thoughts!

PS - I've just read a book by Webb Keane, called 'Ethical Life', and the structure of his book being consciously 1st, 2nd and 3rd person, echoed exactly the above...pointing perhaps in the same direction 

[Jonathan Reams vol_18_no_1_reams_review_of_facing_the_complexity_gap.pdf](https://github.com/life-itself/community/files/13976504/Jonathan.Reams.vol_18_no_1_reams_review_of_facing_the_complexity_gap.pdf)

[Aiden Thornton vol_18_no_1_thornton_response_to_reams_review_of_facing_the_complexity_gap.pdf](https://github.com/life-itself/community/files/13976495/Aiden.Thornton.vol_18_no_1_thornton_response_to_reams_review_of_facing_the_complexity_gap.pdf)

**Reply by asimong** [reply]
Happy to collaborate if that fits, John @baouroux

**Reply by baouroux** [reply]
Thanks Simon...that'd be wonderful...I'm hoping that the above actually can be put into practical action with 'Developmentally' designed signifiers for narratives that arise in dialogues. That is, how we might interpret or signify what are people's constructs when in dialogue/debate. 

We were actually deploying client projects towards the end of my 3+ years of working with Dave Snowden's SenseMaker narratives sensemaking design, using developmental stage signifiers, in leadership 360 feedback platforms. (A couple of years later Dave spoke on  the Rebel Wisdom podcast of looking into using the Kegan stages as 'equally valued modulators' in narratives signification projects, but I don't think anything has progressed on that front AND his notion of modulators vs. stages might be going too far in flattening out the frameworks.)

On the dialogue side, one large client and high stake project (in the context of a conflict between management and union at a factory), we were able to shift gradually from the 'survey' paradigm trap that I felt Dave Snowden's SenseMaker typical projects still fell into (across a broad community internationally using the platform), by creating a far more open access to the narratives data and visualisations for the co-workers, where small action groups were to interpret and build actions from 'bottom-up'. 

I'd like to put both together - i.e. the developmental signification, and the dialogue....maybe getting towards a critical Dialogos that John Vervaeke speaks to?

And this is my tentative rebranding of our narratives work over the years, under my Human-Equity Ltd consulting brand: https://johncoliver.wixsite.com/collective-navigator

Would be great to speak on one of the group calls perhaps, to hear your thoughts and feedback!

------
Moving the thread to this new discussion thread! 
https://github.com/orgs/life-itself/discussions/1017
--------

**Comment by asimong** [comment]
Wondering if this might be of interest here.. https://wiki.simongrant.org/doku.php/d:2024-02-08 just written!
